src/renderer/components/app.tsx

```diff
@@ -69,6 +72,32 @@ export class App extends React.Component {
     whatInput.ask(); // Start to monitor user input device
   }
 
+  async componentDidMount() {
+    console.log("hello");
+    const cluster = getHostedCluster();
+    await eventStore.loadAll();
+    await nodesStore.loadAll();
+    await podsStore.loadAll();
+    eventStore.subscribe();
+    nodesStore.subscribe();
+    podsStore.subscribe();
+
+    reaction(() => this.warningsCount, (count) => {
+      broadcastMessage(`cluster-warning-event-count:${cluster.id}`, count);
+    });
+  }
+
+  @computed
+  get warningsCount() {
+    let warnings = 0;
+    nodesStore.items.forEach(node => {
```

Comment:
Array.prototype.reduce is better for reducing an array to a single value

Reply:
Or even [lodash's sum](https://lodash.com/docs/4.17.15#sum)

Reply:
Fixed.